website/client/src/mixins/spells.js

```diff
@@ -78,7 +78,7 @@ export default {
     async castEnd (target, type) {
       if (!this.$store.state.spellOptions.castingSpell) return null;
       let beforeQuestProgress;
-      if (this.spell.target === 'party') beforeQuestProgress = this.questProgress();
+      if (this.spell.target === 'task') beforeQuestProgress = this.questProgress();
```

Comment:
The checking if the difference already happens.
I checked all spells, currently all spells that invoke boss damage are `task` spells.

Reply:
I understand but in case we change a spell which does not target a task it would be nice to get it to show the boss damage without any code change here